docs/source/markdown/podman-image-unmount.1.md

```diff
@@ -40,4 +40,4 @@ podman image unmount imageID1 imageID2 imageID3
 podman image unmount --all
 
 ## SEE ALSO
-podman(1), podman-image-mount(1), podman-container-mount(1)
+**[podman(1)](podman.1.md)**, **[podman-image-mount(1)](podman-image-mount.1.md)**, **[podman-container-mount(1)](podman-containeri-mount.1.md)**
```

Comment:
Typo ("containeri" with unwanted "i"). Caught by eyeball review, which is really hard, so I would like to (tomorrow) write myself a nice little tool to do an automated double-check.

Reply:
Thanks SGTM.  Going through this was painful, and likely error prone, but it is better then what we had.